src/main/java/pl/damianszczepanik/jenkins/buildhistorymanager/descriptors/actions/DeleteArtifactsWithPatternActionDescriptor.java

```diff
@@ -0,0 +1,47 @@
+package pl.damianszczepanik.jenkins.buildhistorymanager.descriptors.actions;
+
+import org.jenkinsci.Symbol;
+import org.kohsuke.stapler.DataBoundSetter;
+
+import hudson.Extension;
+import hudson.model.Descriptor;
+import pl.damianszczepanik.jenkins.buildhistorymanager.model.actions.Action;
+import pl.damianszczepanik.jenkins.buildhistorymanager.model.actions.DeleteArtifactsWithPatternAction;
+
+/**
+ * Descriptor implementation needed to render UI for {@link DeleteArtifactsWithPatternAction}.
+ */
+@Extension
+@Symbol("DeleteArtifactsWithPattern")
+public class DeleteArtifactsWithPatternActionDescriptor extends Descriptor<Action>{
+
+    private String include;
```

Comment:
this is verb, attribute is a noun, not verb

Reply:
@damianszczepanik @martinda @olivierdagenais 
Please cast your vote for renaming options for the ` include / exclude` attributes:

A) inclusions / exclusions
B) includePattern / excludePattern
C) includePatterns / excludePatterns
B) includedFiles / excludedFiles

Reply:
To me `includePatterns` / `excludePatterns` fits best, because we are expecting those fields to represent:

> (...) a comma-separated list of Ant-style globs (...)

Reply:
The attribute names `include` and `exclude` have been changed to `includePatterns` and `excludePatterns` respectively.